pkg/controller/volume/attachdetach/cache/actual_state_of_world.go

```diff
@@ -487,6 +492,27 @@ func (asw *actualStateOfWorld) addVolumeToReportAsAttached(
 	}
 }
 
+// AddNodeFromAttachUpdates add new node to nodesToUpdateStatusFor
+func (asw *actualStateOfWorld) AddNodeFromAttachUpdates(nodeName types.NodeName) {
+	asw.Lock()
+	defer asw.Unlock()
+	nodeToUpdate, nodeToUpdateExists := asw.nodesToUpdateStatusFor[nodeName]
+	if nodeToUpdateExists {
+		// should not happen
+		errMsg := fmt.Sprintf("Failed to add new node %q to nodesToUpdateStatusFor, "+
```

Comment:
A nodeDelete event will be received when  a node is terminated or recreated, and I have deleted the old node from controller cache at Line 483, so there is no conflict.
Please let me know if my understand is wrong, thank you.